examples.md

```diff
@@ -435,12 +435,16 @@ jobs:
 ## Python - pipenv
 
 ```yaml
+- name: Set up Python
```

Comment:
Could you remove this step?  We don't include the `setup-*` actions in other examples and it won't necessarily always be needed.

Reply:
In order to get the patch version of python into the cache key, the `actions/setup-python` step needs an `id` assigned to it that matches the IDx inside `${{ steps.setup-python.outputs.python-version }}`.

Would you like me to add a comment in the `setup-python` step to make that more clear?

Reply:
Got it.  Yes I think that would be helpful 👍